doc/api/child_process.markdown

```diff
@@ -194,20 +194,22 @@ replace the existing process and uses a shell to execute the command.*
   * `stderr` {Buffer}
 * Return: ChildProcess object
 
-The `child_process.execFile()` method is similar to [`child_process.exec()`][]
-except that it does not first spawn a shell. Rather, the specified `command` is
-spawned directly as a new process making it slightly more efficient than
-[`child_process.exec()`][]. The same options are support by both
-`child_process.exec()` and `child_process.execFile()`.
+The `child_process.execFile()` function is similar to [`child_process.exec()`][]
+except that it does not spawn a shell. Rather, the specified executable `file`
+is spawned directly as a new process making it slightly more efficient than
+[`child_process.exec()`][].
 
-    const exec = require('child_process').execFile;
-    const child = execFile('cat *.js bad_file | wc -l',
-      (error, stdout, stderr) => {
-        console.log(`stdout: ${stdout}`);
-        console.log(`stderr: ${stderr}`);
-        if (error !== null) {
-          console.log(`exec error: ${error}`);
-        }
+The same options as `child_process.exec()` are supported. And given a relative
+`file` path, `child_process.execFile()` searches for a matching executable in
```

Comment:
Not correct. A relative path (such as `../fu`) will *not* be searched for in the `PATH`, that's what the linux docs mean when they say "will not look in PATH if the filename contains a `/`).